pkg/editior/interactive.go

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright 2020 Dell Inc.
```

Comment:
Would you please rename this directory to "editor" versus the current version -- feels like a typo.